sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -54,6 +54,124 @@
 #endif
 
 
+
+/*
+ * S.bus decoder matrix. (extracted from PX4)
```

Comment:
I actually tried what you've done over there before I made this, but it gave me incorrect results... (the received bytes seemed to be constantly changing in loops of 4 if I remember correctly)
Also, the current pprz implementation is quite messy regarding some of the protocol definitions. For example, SBUS_FLAGS_BYTE is set to 22, while it should be 23 (counted from 0). For some reason, pprz does not count the start byte in the package...
That initially really made it difficult to compare and understand what's going on. There is some more stuff like that which I encountered last week but don't remember from the top of my head.

Have you tried this on your original sbus equipment and found it faulty? I can try this again of course, but probably Friday at the earliest.